server/routes/initialAppointment/handlers/hardStop/index.ts

```diff
@@ -47,20 +49,49 @@ export default function Index({ licenceService, conditionService }: Services): R
     get('/create/id/:licenceId/initial-meeting-name', controller.GET)
     post('/create/id/:licenceId/initial-meeting-name', controller.POST, PersonName)
   }
+  {
+    const controller = new InitialMeetingNameRoutes(licenceService)
+    get('/edit/id/:licenceId/initial-meeting-name', controller.GET)
+    post('/edit/id/:licenceId/initial-meeting-name', controller.POST, PersonName)
+  }
   {
     const controller = new InitialMeetingPlaceRoutes(licenceService, UserType.PRISON)
     get('/create/id/:licenceId/initial-meeting-place', controller.GET)
     post('/create/id/:licenceId/initial-meeting-place', controller.POST, Address)
   }
+  {
+    const controller = new InitialMeetingPlaceRoutes(licenceService, UserType.PRISON)
+    get('/edit/id/:licenceId/initial-meeting-place', controller.GET)
+    post('/edit/id/:licenceId/initial-meeting-place', controller.POST, Address)
```

Comment:
if not parameterising these then may be you could do `(create|edit)/id/:licenceId/initial-meeting-place`